modules/stanford_profile_styles/lib/scss/paragraph-types/react_paragraphs/_multirow.scss

```diff
@@ -0,0 +1,144 @@
+@charset 'UTF-8';
```

Comment:
I moved this out of the WYSIWYG type and on to its own as it contains a lot more than just the WYSIWYG styles.

Reply:
That makes sense.